tools/rum/cruncher.js

```diff
@@ -557,9 +557,23 @@ export class DataChunks {
    * checkpoint attribute set to 'click' must exist.
    * @param {string} combiner used to determine if all or some filters must match.
    * By default, 'every' is used.
+   * @param {boolean} isVisiting calculate the conversion of the visiting
+   * set instead of the entire dataset. Useful when calculating the conversion
+   * rate in relation to visits
    * @returns {boolean} the result of the check.
    */
-  hasConversion(aBundle, filterSpec, combiner) {
+  hasConversion(aBundle, filterSpec, combiner, isVisiting = true) {
```

Comment:
Two problems with this:
1. it's better to make this explicit in the conversion spec rather than in the function call
2. this definition would only count conversions that happen right on entering the page. This is the minority of conversions and would make it impossible to deal with things like checkout processes

Reply:
In the current implementation a visit is defined by the existence of an enter checkpoint in the bundle. 

How would that change in the case of checkout processes ?